fix: `print()` multiline behaviour (#17319)
sveltejs/svelte: .changeset/rich-zoos-walk.md

---
'svelte': patch
---

> fix: multiline html blocks now properlery seperated from other multiline blocks while `print()`-ing an AST

Reviewer: ```suggestion
fix: properly separate multiline html blocks from each other in `print()`
```
